src/api/github/getClient.ts

```diff
@@ -43,31 +26,6 @@ export async function getClient(type: ClientType, orgSlug?: string | null) {
         'Must pass orgSlug to `getClient` if getting an app scoped client.'
       );
     }
-
-    const org = GETSENTRY_ORG;
-
-    let client = _CLIENTS_BY_ORG.get(orgSlug);
-    if (client === undefined) {
-      // Bootstrap with a client not bound to an org.
-      const appClient = _getAppClient(org.appAuth);
-
-      // Use the unbound client to hydrate a client bound to an org.
-      const installation = await appClient.apps.getOrgInstallation({
-        org: orgSlug,
-      });
-      org.appAuth.installationId = installation.data.id;
-      client = _getAppClient(org.appAuth);
-
-      // The docs say it's safe for client instances to be long-lived:
-      //
-      // > Additionally, the SDK will take care of regenerating an installation
-      // > access token for you so you don't need to worry about the one hour
-      // > expiration.
-      //
-      // https://docs.github.com/en/apps/creating-github-apps/authenticating-with-a-github-app/authenticating-as-a-github-app-installation#using-the-octokitjs-sdk-to-authenticate-as-an-app-installation
-
-      _CLIENTS_BY_ORG.set(orgSlug, client);
-    }
-    return client;
+    return GETSENTRY_ORG.api; // ☢️
```

Comment:
We move all of the logic for instantiating the `Octokit` client over to `GitHubOrg`, but we return it here so that at this point we don't have to change anything in how `getClient` is called throughout the app.